mitodl/mitxpro
# Jk/hubspot line sync

#### Pre-Flight checklist


#### What are the relevant tickets?
#437 #485 

#### What's this PR do?
Handles LINE_ITEM syncs that fails because their DEALS haven't synced.
Adds sync all management command

#### How should this be manually tested?
Try syncing a line_item before you sync its deal, then sync the deal, then ensure that `check_hubspot_api_errors` resyncs that line without logging an error.
Run the management command

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
